planner/version.go

```diff
@@ -0,0 +1,55 @@
+package planner
```

Comment:
Could you add
```
// Copyright 2022 Jetpack Technologies Inc and contributors. All rights reserved.
// Use of this source code is governed by the license in the LICENSE file.

```

To any new file? (if there's a way we could enforce that via linting, that'd be awesome)